### PR TITLE
Fix missing i18t tokens

### PR DESCRIPTION
**Pull request by jmcameron**
Fixed missing and extra/orphaned translation i18n translation tokens.   Also re-sorted lines in tree.json in alphabetical order.

Output from running **utilities/tfcomp.js**:
```
$ node utilities/translation/tfcomp.js client/src/i18n/en client/src/i18n/fr

Missing from /home/jmcameron/Bhima/client/src/i18n/fr/depot.json:
  DEPOT.ADD_DEPENDENT_DEPOT

Missing from /home/jmcameron/Bhima/client/src/i18n/en/form.json:
  FORM.LABELS.EXTRANCT_EMPLOYEE_STANDING

Missing from /home/jmcameron/Bhima/client/src/i18n/en/install.json:
  INSTALL.HERE

Missing from /home/jmcameron/Bhima/client/src/i18n/en/stock.json:
  STOCK.EXCLUDE_ALL_EXAUSTED_STOCK
  STOCK.INCLUDE_ONLY_EXHAUSTED_STOCK
  STOCK.RECEIPT.EXIT_DEPOTS

Missing from /home/jmcameron/Bhima/client/src/i18n/fr/tree.json:
  TREE.SURVEY_FORM
```

All these errors have been fixed.

### CODE REVIEW

File: client/src/i18n/en/tree.json [hunk 1]
```diff
@@ -103,37 +102,38 @@
...
-    "PURCHASE_ORDER_ANALYSIS": "[Stock] Purchase Order Analysis",
-    "PURCHASE_ORDER" : "Purchase Orders",
     "PURCHASE" : "Purchase",
+    "PURCHASE_ORDER" : "Purchase Orders",
+    "PURCHASE_ORDER_ANALYSIS": "[Stock] Purchase Order Analysis",
     "PURCHASE_REGISTRY" : "Purchase Registry",
     "PURCHASING" : "Purchasing",
     "REALIZED_PROFIT_REPORT": "[Invoicing] Collection on invoices",
-    "REFERENCE_GROUP" : "Reference Group",
+    "RECOVERY_CAPACITY_REPORT": "Recovery Capacity",
```
Comment (mbayopanda) [inline client/src/i18n/en/tree.json:130]: 👍

**Review by mbayopanda — COMMENTED** [review]
This is a good idea 👍 

I have also noticed this : 

image

**Comment by jmcameron** [comment]
Thanks @mbayopanda .   When I run the tfcomp.js, I do not see the CASH.DEBTOR_INVOICES items which is strange.  But when I actually check the files, there is a discrepancy.  There may be a bug in tfcomp.js.  Not sure why it is different on your system vs mine!

**Comment by jmcameron** [comment]
Just fixed the problem with CASH.DEBTOR_INVOICES.TITLE.  
But I still do not see CASH.DEBTOR_INVOICES.1, etc, in the code anywhere, so I'm ignoring those discrepancies.

File: client/src/i18n/en/tree.json [hunk 2]
```diff
@@ -65,36 +64,36 @@
...
     "INVOICING_FEES" : "Invoicing Fees",
     "IPR_MANAGEMENT" : "IPR Management",
     "IPR_TAX_CONFIGURATION" : "IPR Tax Configuration",
     "JOURNAL_VOUCHER" : "Journal Voucher",
     "LOCATION" : "Location Management",
     "MONTHLY_BALANCE": "Monthly Balance",
     "MONTHLY_CONSUMPTION" : "[Stock] Monthly consumption",
-    "MULTI_PAYROLL_INDICE" : "Multiple Payroll(indice)",
     "MULTI_PAYROLL" : "Multiple Payroll",
+    "MULTI_PAYROLL_INDICE" : "Multiple Payroll(indice)",
```
Comment (jniles) [inline client/src/i18n/en/tree.json]: Let's call this "Multiple Payroll (Index-based)".  It's somewhat a long story at this point.

File: client/src/i18n/fr/form.json [hunk 3]
```diff
@@ -451,7 +451,7 @@
             "EXIT": "Sorties",
             "EXPENSE_ACCOUNT": "Comptes de charges",
             "EXPENSE": "Charges",
-            "EXTRANCT_EMPLOYEE_STANDING": "Extrait de la situation financière de l'employée pour la période",
+            "EXTRACT_EMPLOYEE_STANDING": "Extrait de la situation financière de l'employée pour la période",
```
Comment (jniles) [inline client/src/i18n/fr/form.json:454]: Nice :+1:

File: client/src/i18n/fr/stock.json [hunk 4]
```diff
@@ -29,8 +29,6 @@
     "ENTRY_DONATION"  : "Donation",
     "ENTRY_INTEGRATION" : "Intégration",
     "ENTRY_TRANSFER"  : "Transfert",
-    "EXCLUDE_ALL_EXAUSTED_STOCK": "Exclure tous les stocks épuisés",
```
Comment (jniles) [inline client/src/i18n/fr/stock.json:33]: Are you sure these aren't used anywhere?

**Review by jniles — APPROVED** [review]
LGTM.

File: client/src/i18n/fr/stock.json (hunk 4 above)
Comment (jmcameron) [reply client/src/i18n/fr/stock.json:33]: I could not find these in global searches.  So I think they are not used at all.

File: client/src/i18n/en/tree.json (hunk 2 above)
Comment (jmcameron) [reply client/src/i18n/en/tree.json]: Ok, I will fix this.  Don't release the PR until I update it.

Comment (jmcameron) [reply client/src/i18n/en/tree.json]: Done.  Ready to release.

**Comment by jniles** [comment]
I've made an update to bors to skip Travis for the moment.  Let me see if it has taken hold.

bors r+

**Comment by jniles** [comment]
Whoops!  Great minds think alike.

**Comment by jmcameron** [comment]
Looks like Bors is hung?

**Comment by jniles** [comment]
Yeah.  Can you try rebasing this PR?

**Comment by jmcameron** [comment]
> Yeah. Can you try rebasing this PR?

Done.

**Comment by jniles** [comment]
bors r+

Trying again!

**Pull request merged**